src/Microsoft.Health.Dicom.Core.UnitTests/Features/Query/QueryServiceTests.cs

```diff
@@ -54,23 +54,24 @@ public QueryServiceTests()
     }
 
     [Theory]
-    [InlineData(QueryResource.StudySeries, "123.001")]
+    [InlineData(QueryResource.StudySeries, "123.001", Skip = "Enable once UID validation rejects leading zeroes.")]
```

Comment:
This test and the test below were written incorrectly and never asserted anything 😲 They each invoke `Assert.ThrowsAsync` but did not `await` the result. As such, when fixed, they now fail because the `001` in the UID is considered valid. I've skipped these scenarios for now